test/org/redhatcop/util/files/params/jenkins-s2i.params

```diff
@@ -0,0 +1,3 @@
+JENKINS_GIT_URL=https://github.com/etsauer/pipeline-library.git
```

Comment:
I'd suggest this is changed to `JENKINS_GIT_URL=https://github.com/redhat-cop/pipeline-library.git` since that's what should be used once this is merged.

Reply:
yes